tests/web/test_max_rows_enforcer.py

```diff
@@ -0,0 +1,51 @@
+from unittest import mock
+
+import pytest
+
+from snuba.clickhouse.columns import ColumnSet
+from snuba.clickhouse.query import Query
+from snuba.datasets.deletion_settings import DeletionSettings
+from snuba.datasets.storages.storage_key import StorageKey
+from snuba.query.data_source.simple import Table
+from snuba.query.exceptions import MaxRowsEnforcerException
+from snuba.web.delete_query import _enforce_max_rows, construct_condition
+
+from_clause = Table(
+    "search_issues_local_v2",
+    ColumnSet([]),
+    storage_key=StorageKey.SEARCH_ISSUES,
+    # TODO: add allocation policies
+    allocation_policies=[],
+)
+body = {
+    "columns": {
+        "occurrence_id": ["d3096a0a-290c-446d-8b01-c37b5c17169b"],
+        "project_id": [3],
+    },
+    "debug": True,
+    "tenant_ids": {"referrer": "test", "organization_id": 1},
+}
+condition = construct_condition(body)
+query = Query(
+    from_clause=from_clause,
+    condition=condition,
+    on_cluster=None,
+    is_delete=True,
+)
+
+
+@pytest.mark.clickhouse_db
+def test_max_row_enforcer_passes() -> None:
+    _enforce_max_rows(query)
```

Comment:
This test doesn't assert anything, I think it might make more sense to have this function return true or false so we can assert that it passes.